openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -772,7 +772,7 @@ port_INLINE bool ieee154e_processIEs(OpenQueueEntry_t* pkt, uint16_t* lenIE) {
         infer the asnOffset based on the fact that
         ieee154e_vars.freq = 11 + (asnOffset + channelOffset)%16 
         */
-        for (i=0;i<16;i++){
+        for (i=0;i<TOTAL_CHANNEL_USED;i++){

```

Comment:
This name is wrong. `TOTAL_CHANNEL_USED` means that you use a channel totally, which doesn't mean anything.
Please replace by `NUM_CHANNELS`.